transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -620,9 +620,11 @@ jint JNI_OnLoad_netty_transport_native_epoll(JavaVM* vm, void* reserved) {
     return ret;
 }
 
+#ifndef NETTY_NOT_DYNAMIC
```

Comment:
would it be possible to avoid double negatives? `if not defined netty not dynamic` -> `if defined netty dynamic` -> `#ifdef NETTY_DYNAMIC_LINK`?